config/config.json

```diff
@@ -9,45 +9,174 @@
+        "cppcoreguidelines-no-malloc" : ["default", "sensitive", "extreme"],
+        "cppcoreguidelines-pro-type-const-cast" : ["extreme"],
+        "cppcoreguidelines-pro-type-member-init" : ["extreme"],
+        "cppcoreguidelines-pro-type-reinterpret-cast" : ["extreme"],
+        "cppcoreguidelines-pro-type-static-cast-downcast" : ["default", "sensitive", "extreme"],
+        "cppcoreguidelines-pro-type-union-access" : ["extreme"],
+        "cppcoreguidelines-pro-type-vararg" : ["sensitive", "extreme"],
+        "cppcoreguidelines-slicing" : ["default", "sensitive", "extreme"],
+        "cppcoreguidelines-special-member-functions" : ["default", "sensitive", "extreme"],
+        "misc-argument-comment" : ["default", "sensitive", "extreme"],
+        "misc-assert-side-effect" : ["default", "sensitive", "extreme"],
+        "misc-bool-pointer-implicit-conversion" : ["default", "sensitive", "extreme"],
+        "misc-dangling-handle" : ["default", "sensitive", "extreme"],
+        "misc-definitions-in-headers" : ["default", "sensitive", "extreme"],
+        "misc-fold-init-type" : ["default", "sensitive", "extreme"],
+        "misc-forward-declaration-namespace" : ["default", "sensitive", "extreme"],
+        "misc-forwarding-reference-overload" : ["default", "sensitive", "extreme"],
+        "misc-inaccurate-erase" : ["default", "sensitive", "extreme"],
+        "misc-incorrect-roundings" : ["default", "sensitive", "extreme"],
+        "misc-inefficient-algorithm" : ["default", "sensitive", "extreme"],
+        "misc-lambda-function-name" : ["default", "sensitive", "extreme"],
+        "misc-macro-parentheses" : ["extreme"],
+        "misc-macro-repeated-side-effects" : ["default", "sensitive", "extreme"],
+        "misc-misplaced-const" : ["default", "sensitive", "extreme"],
+        "misc-misplaced-widening-cast" : ["default", "sensitive", "extreme"],
+        "misc-move-const-arg" : ["default", "sensitive", "extreme"],
+        "misc-move-constructor-init" : ["default", "sensitive", "extreme"],
+        "misc-move-forwarding-reference" : ["default", "sensitive", "extreme"],
+        "misc-multiple-statement-macro" : ["default", "sensitive", "extreme"],
+        "misc-new-delete-overloads" : ["default", "sensitive", "extreme"],
+        "misc-noexcept-move-constructor" : ["default", "sensitive", "extreme"],
+        "misc-non-copyable-objects" : ["default", "sensitive", "extreme"],
+        "misc-redundant-expression" : ["default", "sensitive", "extreme"],
+        "misc-sizeof-container" : ["default", "sensitive", "extreme"],
+        "misc-sizeof-expression" : ["default", "sensitive", "extreme"],
+        "misc-static-assert" : ["default", "sensitive", "extreme"],
+        "misc-string-compare" : ["default", "sensitive", "extreme"],
+        "misc-string-constructor" : ["default", "sensitive", "extreme"],
+        "misc-string-integer-assignment" : ["default", "sensitive", "extreme"],
```

Comment:
This check has some known false positives that I did not have the time to fix yet. I would not turn this on for now.